lib/web/ui/app.go

```diff
@@ -68,6 +69,10 @@ func MakeApps(c MakeAppsConfig) []App {
 		fqdn := AssembleAppFQDN(c.LocalClusterName, c.LocalProxyDNSName, c.AppClusterName, teleApp)
 		labels := []Label{}
 		for name, value := range teleApp.GetAllLabels() {
+			if strings.HasPrefix(name, "teleport.internal/") {
```

Comment:
We should not be hardcoding string `teleport.internal/` verbatim everywhere. There should be a constant for it.